oeps/architectural-decisions/oep-XXXX-micro-frontend-domains.rst

```diff
@@ -0,0 +1,171 @@
+================================
+OEP-XXXX: Micro-frontend Domains
+================================
+
+.. list-table::
+   :widths: 25 75
+
+   * - OEP
+     - :doc:`OEP-XXXX <oep-XXXX-micro-frontend-domains>`
+   * - Title
+     - Micro-frontend Domains
+   * - Last Modified
+     - 2022-11-09
+   * - Authors
+     - Adolfo R. Brandes <adolfo@tcril.org>
+   * - Arbiter
+     - TBD
+   * - Status
+     - Draft
+   * - Type
+     - Architecture
+   * - Created
+     - 2022-11-09
+   * - Review Period
+     - TBD
+   * - Resolution
+     - TBD
+   * - References
+     - TBD
+
+
+********
+Abstract
+********
+
+The decision to gradually implement a micro-frontend architecture for Open edX was a largely successful attempt at avoiding some limitations of the monolithic ``edx-platform`` codebase.  In particular, it allowed the project to integrate more and better features by leveraging newer frontend technology, all much faster than would have been possible otherwise.  At the same time, because features encapsulated in micro-frontends (henceforth, "MFEs") are very loosely coupled to each other, this architecture eliminates most integration bottlenecks, allowing individual teams working on any given feature to not be a lot more *productive*.
```

Comment:
nit: Is the "not" here intended? Wouldn't the loose coupling between MFEs, eliminating most integration bottlenecks, would allow teams to be productive, but "allowing individual teams working on any given feature to **not** be a lot more *productive*." makes it sound like the loose coupling is responsible for a loss of productivity.